proxy/proxy.go

```diff
@@ -7,38 +7,38 @@ import (
 
 	"github.com/9seconds/httransform"
 
-	"github.com/scrapinghub/crawlera-headless-proxy/config"
-	"github.com/scrapinghub/crawlera-headless-proxy/layers"
-	"github.com/scrapinghub/crawlera-headless-proxy/stats"
+	"github.com/zytedata/zyte-headless-proxy/config"
+	"github.com/zytedata/zyte-headless-proxy/layers"
+	"github.com/zytedata/zyte-headless-proxy/stats"
 )
 
 func NewProxy(conf *config.Config, statsContainer *stats.Stats) (*httransform.Server, error) {
-	crawleraURL, err := url.Parse(conf.CrawleraURL())
+	smpURL, err := url.Parse(conf.SmartProxyManagerURL())
 	if err != nil {
-		return nil, fmt.Errorf("incorrect crawlera url: %w", err)
+		return nil, fmt.Errorf("incorrect zyte smart proxy manager url: %w", err)
 	}
 
-	executor, err := httransform.MakeProxyChainExecutor(crawleraURL)
+	executor, err := httransform.MakeProxyChainExecutor(smpURL)
 	if err != nil {
 		return nil, fmt.Errorf("cannot make proxy chain executor: %w", err)
 	}
 
-	crawleraExecutor := func(state *httransform.LayerState) {
+	smpExecutor := func(state *httransform.LayerState) {
 		startTime := time.Now()
 
 		executor(state)
-		statsContainer.NewCrawleraTime(time.Since(startTime))
-		statsContainer.NewCrawleraRequest()
+		statsContainer.NewSmartProxyManagerTime(time.Since(startTime))
+		statsContainer.NewSmartProxyManagerRequest()
 	}
 
 	opts := httransform.ServerOpts{
 		CertCA:           []byte(conf.TLSCaCertificate),
 		CertKey:          []byte(conf.TLSPrivateKey),
-		Executor:         crawleraExecutor,
+		Executor:         smpExecutor,
 		Logger:           &Logger{},
 		Metrics:          statsContainer,
-		Layers:           makeProxyLayers(conf, crawleraExecutor, statsContainer),
-		OrganizationName: "ScrapingHub",
+		Layers:           makeProxyLayers(conf, smpExecutor, statsContainer),
+		OrganizationName: "Zyte",
```

Comment:
It might requires extra changes on proxy with support new certificate. Need double check with OU=Zyte.